### ISSUE TITLE
Separate input / output and "initialization" data ?

### ISSUE THREAD

**Issue by j3r3m1** [orbisgis/geoclimate#559]
When we run geoclimate, we get all files in a same folder (or database). All files actually means:

- input data (building, block, road, etc.): Geoclimate generic input tables produced by the workflow 
- output data (building_indicators, etc.): Geoclimate generic output tables produced by the workflow 
- initialization data files (sea_land_mask, building_height_missing, urban_areas): files which are created during the initialization step and which are dataset dependent (only created when using OSM for now it seems).

I am currently working on the documentation and I think it could be better if we can make the code consistent with the documentation (I mean separating those files in different subfolders).

@franetibe @ebocher @ELSW56 What do you think ?

I would use input data, output data and initialization data as names for the subfolders but I am OK if you have a better solution for the last one...

**Comment by ebocher** [comment]
We need a meeting with a beer to discuss

**Comment by ebocher** [comment]
>     * initialization data files (sea_land_mask, building_height_missing, urban_areas): files which are created during the initialization step and which are dataset dependent (only created when using OSM for now it seems).

Yes currently but we must in the future integrate those tables as optional abstract table because they are used to improve LCZ classifications and other indicators.
